src/lib/sql/sql_translator.cpp

```diff
@@ -144,18 +142,57 @@ std::shared_ptr<AbstractLQPNode> SQLTranslator::_translate_insert(const hsql::In
     current_result_node = std::make_shared<DummyTableNode>();
   }
 
+  // Lambda to compare a DataType to the type of an hqsl::Expr
+  auto literal_matches_data_type = [](const hsql::Expr& expr, const DataType& column_type) {
```

Comment:
Not sure what happened here, but this does not look like it belongs to this PR.

Reply:
Yes. Strangely enough, this is already in the master. I had to copy because the code wouldn’t compile anymore after merging the master.

Reply:
Have a look: https://github.com/hyrise/hyrise/blob/master/src/lib/sql/sql_translator.cpp

Reply:
watt? this is seriously strange

Reply:
I will delete this PR and reopen it.